docs/rbac/roles.rst

```diff
@@ -0,0 +1,275 @@
+        "pulp_created": "2022-05-27T15:27:00.623004Z",
+        "role": "container.containerdistribution_consumer",
+        "content_object": "/pulp/api/v3/distributions/container/container/5b8ec13c-d578-4b3a-9b99-80986e5e00b6/"
+      }
+    ]
+
+
+Collaborator Role
+~~~~~~~~~~~~~~~~~
+
+The *Collaborator* role represents a set of permissions that a co-worker working within a same user-space
+should have. In comparison to the *Consumer* role, users with the *Collaborator* role are allowed
+to add (push) and modify content. The following set of permissions is evaluated for the *Collaborator*
+role for distributions:
+
+.. code-block:: bash
+
+    pulp role show --name "container.containerdistribution_collaborator"
+
+::
+
+    {
+      "pulp_href": "/pulp/api/v3/roles/933e0376-8945-489a-93a6-cafb6753f4bb/",
+      "pulp_created": "2022-05-26T12:02:28.924330Z",
+      "name": "container.containerdistribution_collaborator",
+      "description": null,
+      "permissions": [
+        "container.pull_containerdistribution",
+        "container.push_containerdistribution",
+        "container.view_containerdistribution"
+      ],
+      "locked": true
+    }
+
+
+.. code-block:: bash
+
+    pulp role show --name "container.containernamespace_collaborator"
+
+::
+
+    {
+      "pulp_href": "/pulp/api/v3/roles/1466e614-73a7-4a58-ab36-ced0ab1a1809/",
+      "pulp_created": "2022-05-26T12:02:29.058226Z",
+      "name": "container.containernamespace_collaborator",
+      "description": null,
+      "permissions": [
+        "container.namespace_add_containerdistribution",
+        "container.namespace_change_containerdistribution",
+        "container.namespace_change_containerpushrepository",
+        "container.namespace_delete_containerdistribution",
+        "container.namespace_modify_content_containerpushrepository",
+        "container.namespace_pull_containerdistribution",
+        "container.namespace_push_containerdistribution",
+        "container.namespace_view_containerdistribution",
+        "container.namespace_view_containerpushrepository",
+        "container.view_containernamespace"
+      ],
+      "locked": true
+    }
+
+
+Customizing Access Policies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The plugin is shipped with default access policies that can be modified to achieve different RBAC
+behaviour. This can be done by issuing the following commands:
+
+.. code-block:: bash
+
+    pulp role create --name "container.containerrepository_syncer" \
+        --permission "container.view_containerrepository" \
+        --permission "container.view_containerremote" \
+        --permission "container.change_containerrepository" \
+        --permission "container.modify_content_containerrepository" \
+        --permission "container.sync_containerrepository"
+
+    pulp user role-assignment add --username "alice" --role "container.containerrepository_syncer" object ""
```

Comment:
Should this be an example about changing the AP?

Reply:
Of course. I provide an example of how to create, assign, and hook a role.

For the sake of completeness, I will add an example of assigning default permissions to a user in one of the descriptions of the default roles.

Reply:
I would still consider these two commands as an example to create and use a "custom role". "changing the access policy" to me is the next level of customization. My suggestion: Make this example its own thing, and move the headline for AP below it.

Reply:
sounds good!